frontend/src/console/components/AlertMessage/styles.scss

```diff
@@ -0,0 +1,8 @@
+@import '~styles/theme';
+
+.alert-warning {
+  background-color: #6d9cae;
+  border-radius: 15px;
+  border: 0px;
+  color: white;
+}
```

Comment:
Don't hard-code colour or font values.  If you want to use this colour, introduce a new variable for it. I think it's better to use the default bootstrap theme colour. If that needs to be changed, change it at styles/_theme.scss.